### PR TITLE
Adds featured-sponsors-darkmode.svg to fix darkmode sponsors on ddev/ddev

### PR DESCRIPTION
**Pull request by bmartinez287**
Adds featured-sponsors-darkmode.svg to fix dark mode sponsors on ddev/ddev

## The Issue
https://github.com/ddev/ddev/issues/5492

## How This PR Solves The Issue
This PR makes the asset available

### CODE REVIEW

**Comment by bmartinez287** [comment]
https://35a114ba.ddev-com-front-end.pages.dev/resources/featured-sponsors-darkmode.svg
https://35a114ba.ddev-com-front-end.pages.dev/resources/featured-sponsors.svg

**Review by rfay — CHANGES_REQUESTED** [review]
Well, it certainly looks fine, but how did you recreate the problem logos? That needs to be added to the README I guess unless you're permanently available for this task.

@mattstein thoughts?

**Comment by bmartinez287** [comment]
I basically used the same method I used on ddev.com on this. Most of the logos already existed except for a few that I was just inverting the colors.
The one that proved most challenging was Drupaleasy since it's a PNG but I got it done.

I use inkscape for all but drupaleasy to edit the SVG's then select all the letters icons and on the color pallete switch them to pure white and export the file with a new name. Inkscape is an open-source tool but any other's like illustrator can accomplish the same goal.

**Comment by bmartinez287** [comment]
I can be on the watch for those PRs and add them as needed. It takes a min or two if I have a laptop with me.

**Comment by rfay** [comment]
It's never a rush. Thanks for your help with it. Could just turn the new-sponsor task over to you as well.

**Comment by bmartinez287** [comment]
Yeah, I'm down. If you get the files and share them as an issue I can add the PR for them

**Review by mattstein — CHANGES_REQUESTED** [review]
I didn’t realize you could use a `<picture>` element in a GitHub readme to solve the problem—cool!

I initially shied away from dark mode over logo management, but now that we’re here it might be worth standardizing the sponsor intake process and having a shared source for these so it’s not dependent on any one person hammering them out. I have a pile of logos in Figma I cleaned up for that first push, but I’m not sure what the most convenient place would be.

If we end up with clean, organization-approved SVG files then maybe it’s not a big deal. 🤷‍♂️

File: src/pages/resources/featured-sponsors-darkmode.svg.js
[the remark this answers is not in the document]
Comment (bmartinez287) [reply src/pages/resources/featured-sponsors-darkmode.svg.js:1]: mmm, that's an interesting idea. I left it like that for simplicity but I can see why that could be more maintainable. I will take a look and see what can be done with partials here.

**Pull request merged**